airbyte-integrations/connectors/destination-bigquery/Dockerfile

```
@@ -25,6 +25,8 @@ WORKDIR /airbyte/normalization_code
 RUN pip3 install .
 WORKDIR /airbyte/normalization_code/dbt-template/
 # Download external dbt dependencies
+# amazon linux 2 isn't compatible with urllib3 2.x, so force 1.26.15
+RUN pip3 install urllib3==1.26.15
```

Hello, urllib3 maintainer here :wave: 

Please consider the following so that you continue getting urllib3 1.26 security updates

```suggestion
RUN pip3 install urllib3<2
```

---

good call, thanks!